### PR TITLE
My side of the story

### PR DESCRIPTION
**Pull request by Ozuitoku66**

### CODE REVIEW

File: fairytale.md [hunk 1]
```diff
@@ -15,4 +15,9 @@ The average Squonk is 7 meters tall and screams loud.
 >  What does the **world** he lives in **look like**?
 >  What happened between it and the *Drowsloks*?
 
-`Well... your turn now.`
+I think Squonks do not exist!
```
Comment (floridop) [inline fairytale.md:18]: Hi @Ozuitoku66  ! Could you please move your story above the line
```
>  Add some lines to the story. Can be anything. Inspiration?  
```
that is, around line 10?

More info will come later.

**Comment by floridop** [comment]
Hi @Ozuitoku66  ! Thanks for your submission.

Unfortunately it seems that there were changes in the master that conflict with yours.
Could you perform a merge as described in Tutorial 5 slides 84-96 and integrate those changes?
We do not want to lose any of them, so you can keep what others have written and add yours.
You're free to reorganize the content so that it still makes a story, but please do not modify
the content of what others have written.
You have time until the 6th of November at 19:30. If you can't make it by that time please let me know asap.

**Comment by Ozuitoku66** [comment]
Yes, I will Mr Florido!

**Comment by Ozuitoku66** [comment]
I made the merge and solve the conflict by "sewing" the fairytale as well, Mr Florido.
Did I fulfil your request?

File: fairytale.md [hunk 2]
```diff
@@ -15,4 +20,4 @@ The average Squonk is 7 meters tall and screams loud.
 >  What does the **world** he lives in **look like**?
 >  What happened between it and the *Drowsloks*?
 
-`Well... your turn now.`
```
Comment (floridop) [inline fairytale.md:18]: Why is this line removed? It is present in my master. Something went wrong with the merge. Could you add back?

**Comment by floridop** [comment]
Unfortunately something must have gone wrong, if the merge is successful you should see a green button and a message that the merge is possible above. I am not sure about what happened.

**Comment by floridop** [comment]
Checking the logs, you did not perform the merge as described in the slides yet. I suggest you restore the missing line at https://github.com/floridop/MNXB01-fairytale/pull/12#discussion_r518233461 and then follow the instructions in slides 84-96.

**Comment by Ozuitoku66** [comment]
I will do it! ^_^

**Comment by Ozuitoku66** [comment]
@floridop I got a notification that I successfully completed the merge.

**Comment by floridop** [comment]
Grade updated on canvas
